### ISSUE TITLE
[QA] eos trashbin not accessible in WEB UI

### ISSUE THREAD

**Issue by jnweiger** [owncloud/ocis#863]
Setup:
* docker-compose-eos-test.yml
* enable trashbin via
```
eos space config default space.policy.recycle=on
eos recycle config --add-bin /eos/dockertest/reva/users
eos recycle config --size 1G
```

Reproduce:
* User marie deletes a file `crypt-o.mp3`
* We can see the file in the eos trashcan
```
eos -r 0 0 recycle ls -g
# Deletion Time            UID      GID      SIZE         TYPE          RESTORE-KEY          RESTORE-PATH
# ==============================================================================================================================
Mon Nov 16 14:07:26 2020   20001    30000    1583104      file          fxid:0000000000000009 /eos/dockertest/reva/users/f/f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c/crypt-o.mp3
Mon Nov 16 14:07:26 2020   20001    30000    0            recursive-dir pxid:0000000000000020 /eos/dockertest/reva/users/f/f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c/.sys.v#.crypt-o.mp3/
```

When marie visits the deleted files, there is an error:
![image](https://user-images.githubusercontent.com/1108546/99267164-e1ab5300-2823-11eb-9ebf-6e80c6edb7be.png)
![image](https://user-images.githubusercontent.com/1108546/99267972-e1f81e00-2824-11eb-97d7-41ca3e14ec85.png)

Trying the failing request with curl:
```
$ curl 'https://localhost:9200/remote.php/dav/trash-bin/marie/' -X PROPFIND -H 'User-Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:82.0) Gecko/20100101 Firefox/82.0' -H 'Accept: */*' -H 'Accept-Language: en-US,en;q=0.5' --compressed -H 'X-Requested-With: XMLHttpRequest' -H 'OCS-APIREQUEST: true' -H 'authorization: Bearer eyJhbGciOiJQUzI1NiIsImtpZCI6IiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJwaG9lbml4IiwiZXhwIjoxNjA1NTM4NTg2LCJqdGkiOiJ0eEt1SUE4X0E2WEhob2JZdjNwd0pEUWZCWjVhZFQzaCIsImlhdCI6MTYwNTUzNzk4NiwiaXNzIjoiaHR0cHM6Ly9sb2NhbGhvc3Q6OTIwMCIsInN1YiI6ImxlV1I2Q3RZdEpfQWZEbVBGWk5YYUZQZHJCbnZaZU9SZ2p1dlYzUlcyRzM5VDV3bDdtaFZNVzVVRXRza1NRU2N6eGlwY1R5UGZDQzdqZ25vVG1ObTVBQGtvbm5lY3QiLCJrYy5pc0FjY2Vzc1Rva2VuIjp0cnVlLCJrYy5hdXRob3JpemVkU2NvcGVzIjpbIm9wZW5pZCIsInByb2ZpbGUiLCJlbWFpbCJdLCJrYy5pZGVudGl0eSI6eyJrYy5pLmRuIjoibWFyaWUiLCJrYy5pLmlkIjoiY249bWFyaWUsb3U9dXNlcnMsZGM9ZXhhbXBsZSxkYz1vcmciLCJrYy5pLnVuIjoibWFyaWUifSwia2MucHJvdmlkZXIiOiJpZGVudGlmaWVyLWxkYXAifQ.imXnSHShkMJUx8ULO41zGaB-Bly50B4YrR9_vVfpe9qB-TNcH3qIUaEX4KSKqEzqh0CLhTV2c5ZUVJWvT5ikA9Lc_4W1e9J2hYDXW_Doxs6y-cQI1WC-iodRyi9tkyvzpLiR91HGd4dkro_GpR1x1XbihjEuUN27RVXlDU5hf6FplNwf1Nx-rJ51cNMcIpzC5hxTPHnK1WaEXeOUAvWHZo-3RWYPA9pyNmb_ryqRQhMHhsvsVLk5kOJVPfBfoiJ5T9SYcrceK6HM65416QVKh3oJiWIOCcklMCFzLz3DE9AxyM_9AIEo6GA06MWzlaPbmf28MIDN0tfv4C7ycBVk5A' -H 'Depth: 1' -H 'Content-Type: application/xml; charset=utf-8' -H 'Origin: https://localhost:9200' -H 'Connection: keep-alive' -H 'Referer: https://localhost:9200/' -H 'Pragma: no-cache' -H 'Cache-Control: no-cache' -k 
$ curl 'https://localhost:9200/remote.php/dav/trash-bin/f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c/' -X PROPFIND -H 'User-Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:82.0) Gecko/20100101 Firefox/82.0' -H 'Accept: */*' -H 'Accept-Language: en-US,en;q=0.5' --compressed -H 'X-Requested-With: XMLHttpRequest' -H 'OCS-APIREQUEST: true' -H 'authorization: Bearer eyJhbGciOiJQUzI1NiIsImtpZCI6IiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJwaG9lbml4IiwiZXhwIjoxNjA1NTM4NTg2LCJqdGkiOiJ0eEt1SUE4X0E2WEhob2JZdjNwd0pEUWZCWjVhZFQzaCIsImlhdCI6MTYwNTUzNzk4NiwiaXNzIjoiaHR0cHM6Ly9sb2NhbGhvc3Q6OTIwMCIsInN1YiI6ImxlV1I2Q3RZdEpfQWZEbVBGWk5YYUZQZHJCbnZaZU9SZ2p1dlYzUlcyRzM5VDV3bDdtaFZNVzVVRXRza1NRU2N6eGlwY1R5UGZDQzdqZ25vVG1ObTVBQGtvbm5lY3QiLCJrYy5pc0FjY2Vzc1Rva2VuIjp0cnVlLCJrYy5hdXRob3JpemVkU2NvcGVzIjpbIm9wZW5pZCIsInByb2ZpbGUiLCJlbWFpbCJdLCJrYy5pZGVudGl0eSI6eyJrYy5pLmRuIjoibWFyaWUiLCJrYy5pLmlkIjoiY249bWFyaWUsb3U9dXNlcnMsZGM9ZXhhbXBsZSxkYz1vcmciLCJrYy5pLnVuIjoibWFyaWUifSwia2MucHJvdmlkZXIiOiJpZGVudGlmaWVyLWxkYXAifQ.imXnSHShkMJUx8ULO41zGaB-Bly50B4YrR9_vVfpe9qB-TNcH3qIUaEX4KSKqEzqh0CLhTV2c5ZUVJWvT5ikA9Lc_4W1e9J2hYDXW_Doxs6y-cQI1WC-iodRyi9tkyvzpLiR91HGd4dkro_GpR1x1XbihjEuUN27RVXlDU5hf6FplNwf1Nx-rJ51cNMcIpzC5hxTPHnK1WaEXeOUAvWHZo-3RWYPA9pyNmb_ryqRQhMHhsvsVLk5kOJVPfBfoiJ5T9SYcrceK6HM65416QVKh3oJiWIOCcklMCFzLz3DE9AxyM_9AIEo6GA06MWzlaPbmf28MIDN0tfv4C7ycBVk5A' -H 'Depth: 1' -H 'Content-Type: application/xml; charset=utf-8' -H 'Origin: https://localhost:9200' -H 'Connection: keep-alive' -H 'Referer: https://localhost:9200/' -H 'Pragma: no-cache' -H 'Cache-Control: no-cache' -k 
<?xml version="1.0" encoding="utf-8"?><d:multistatus xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns" xmlns:oc="http://owncloud.org/ns"><d:response><d:href>/remote.php/dav/trash-bin/</d:href><d:propstat><d:prop><d:resourcetype><d:collection/></d:resourcetype></d:prop><d:status>HTTP/1.1 200 OK</d:status></d:propstat><d:propstat><d:prop><oc:trashbin-original-filename></oc:trashbin-original-filename><oc:trashbin-original-location></oc:trashbin-original-location><oc:trashbin-delete-datetime></oc:trashbin-delete-datetime><d:getcontentlength></d:getcontentlength></d:prop><d:status>HTTP/1.1 404 Not Found</d:status></d:propstat></d:response><d:response><d:href>/remote.php/dav/trash-bin/fxid:0000000000000009</d:href><d:propstat><d:prop><oc:trashbin-original-filename>crypt-o.mp3</oc:trashbin-original-filename><oc:trashbin-original-location>crypt-o.mp3</oc:trashbin-original-location><oc:trashbin-delete-datetime>Mon, 19 Jan 1970 13:58:55 +0000</oc:trashbin-delete-datetime><d:resourcetype></d:resourcetype><d:getcontentlength>1583104</d:getcontentlength></d:prop><d:status>HTTP/1.1 200 OK</d:status></d:propstat></d:response></d:multistatus>
$
```
When trying marie's user-id instead of the username, then we get a response.
This is the old api. A username should work.

**Comment by micbar** [comment]
enable trashbin should be fixed in newly pushed eos images
I pushed new eos-base and eos-stack images.

Will need an rc4 to get into your stack.

**Comment by micbar** [comment]
@butonic old friend? userid vs username?

**Comment by jnweiger** [comment]
retested with RC5 localhost setup: unchanged.

**Comment by micbar** [comment]
that was expected. Not fixed in RC5. pending

**Comment by micbar** [comment]
@butonic Is that part of the reva update?

**Comment by C0rby** [comment]
@micbar, I don't think so. I just implemented a fix and will open a PR in reva.

**Comment by micbar** [comment]
needs to be fixed before 1.0.0 -> please @C0rby Ping CERN for merge

**Comment by C0rby** [comment]
The PR was merged!
